Flutter/calendar/timeslot-views.md

```diff
@@ -530,6 +530,31 @@ Widget build(BuildContext context) {
 
 ![Allday panel Background color](images/getting-started/alldayPanelColor.jpg)
 
+## Number of days in view
+You can customize the days count, by setting the [numberOfDaysInView]() property of `timeSlotViewSettings` in the calendar.
+
+{% tabs %}
+{% highlight Dart %}
+
+@override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+          body: SfCalendar(
+        view: CalendarView.week,
+        timeSlotViewSettings: const TimeSlotViewSettings(numberOfDaysInView: 3),
+      )),
+    );
+  }
+
+{% endhighlight %}
+{% endtabs %}
+
+![Number of days in view](images/timeslot-views/numberOfDaysInView.jpg)
+
+>**NOTE**
+* It is applicable for day, week, workweek, timeline day, timeline week and timeline workweek views.
```

Comment:
It is applicable for day, week, workweek, timeline day, timeline week, and timeline workweek views.

Reply:
I have updated this in my latest commit